lib/smart_proxy_pulp_plugin/disk_usage.rb

```diff
@@ -0,0 +1,25 @@
+module PulpProxy
+  class DiskUsage
+    attr_accessor :free, :total, :block_device, :path
```

Comment:
maybe consider accepting a unit argument, so you can decide if its bytes, kilo, mb, gb or human